frontend/src/pages/create_campaign/Campaign/index.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { FormControlLabel, Switch } from "@mui/material";
+import PropTypes from "prop-types";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import LocalizationProvider from "@mui/lab/LocalizationProvider";
+import DateTimePicker from "@mui/lab/DateTimePicker";
+import Dropzone from "react-dropzone";
+import {
+  CampaignContainer,
+  CampaignDropzone,
+  CampaignRowDiv,
+  SwitchRowDiv,
+  CampaignTextField,
+} from "./campaignTab.styled";
+import { fileToDataUrl } from "../../../utils";
+
+const CampaignTab = ({ isSelected, campaignData }) => {
+  const {
+    campaignName,
+    setCampaignName,
+    startDate,
+    setStartDate,
+    endDate,
+    setEndDate,
+    description,
+    setDescription,
+    interviewStage,
+    setInterviewStage,
+    scoringStage,
+    setScoringStage,
+    cover,
+    setCover,
+  } = campaignData;
```

Comment:
i'd be down for `campaign` as the variable name, since its like the campaign object